src/generateCanonicalFieldsInserts.ts

```diff
@@ -28,16 +28,27 @@ psvInput.pipe(
     delimiter: '|',
     asObject: true,
   }),
-).on('data', (row: any) => {
-  const label = row.label.replace('\'', '\'\'');
-  const shortCode = row.shortCode.replace('\'', '\'\'');
-  const dataType = row.dataType.replace('\'', '\'\'');
-  const createdAt = row.createdAt.replace('\'', '\'\'');
+).on('data', (row: Object) => {
+  const record = Object.values(row)[0];
+  //regex to remove extra quotes and correct quotes hierarchy i.e. \" inside datatype string enclosed by \' 
+  const string = record.replace(/\"/g,'\'').replace(/\'\'/g,'\"')
```

Comment:
At first I was not sure this was needed, but I think something like this is needed for `Type`, right?